backend/de.metas.swat/de.metas.swat.base/src/main/sql/postgresql/system/47-de.metas.invoicecandidate/57262309_sys_fix_C_Invoice_Candidate_HeaderAggregation_InvoicingGroupNo_ft.sql

```diff
@@ -0,0 +1,42 @@
+-- this DDL was not found in any migration-script
+-- adding it now
+-- the change is to extend the second generate_series parameter to 10000000 
+CREATE OR REPLACE FUNCTION C_Invoice_Candidate_HeaderAggregation_InvoicingGroupNo_ft()
```

Comment:
pls also update/introduce the DDL file

Reply:
hi @teosarca the ddl file was there all along
what i did now was copy the exisitng DDL into this new migration-script-file

Reply:
@teosarca FYI: https://github.com/metasfresh/metasfresh/blob/big_coconut_uat/backend/de.metas.swat/de.metas.swat.base/src/main/sql/postgresql/ddl/functions/C_Invoice_Candidate_HeaderAggregation_InvoicingGroupNo_ft.sql